app/views/agent/partials/optOutForMTDVATPartial.scala.html

```diff
@@ -0,0 +1,39 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import common.MandationStatus
+
+@(mandationStatus: String)(implicit messages: Messages, appConfig: AppConfig)
+
+@OptOutCard = {
+<div class="card">
+
+    <div class="card-body">
+        <h2 class="heading-medium">
+            <a href="@{appConfig.optOutMtdVatUrl}" >@messages("optOutCard.heading")</a>
+        </h2>
+        <p>@messages("optOutCard.content")</p>
+    </div>
+</div>
+}
+
+@{
```

Comment:
This logic doesn't cater for Voluntary users. It needs to be:
If non-mtdfb -> show nothing
else -> show opt out partial
This way, both 'mandated' and 'voluntary' users see the same opt out link